### PR TITLE
Create 2017-05-22-Sherlock-Holmes-Day.md

### PR DESCRIPTION
**Pull request by mattsaunby**
### Blog title
Sherlock Holmes Day
### Blog description
A short blog about Testers investigation techniques etc to be posted on 22nd May for Sherlock Holmes Day
#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2017-05-22-Sherlock-Holmes-Day.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+categories: testers
+author: mattsaunby
+tags: testers qa investigation sherlock holmes
+comments: true
+excerpt: "On the birthday of Sir Arthur Conan Doyle, we pry into the minds of our Software Testers and how they approach investigation tasks."
+---
+
+Unofficially, 22nd May is Sherlock Holmes day and so we asked some of our Software Testers about investigations, something that Testers here at Holiday Extras do all the time.
+
+The first thing we asked was whether any tools are ever used to help investigations. We received a broad spectrum of answers including Cross Browser Testing, Recordit, Google Analytics and Chrome Dev Tools as well as various Chrome Extensions. The most popular answer however, was the knowledge of fellow testers, as well as your own knowledge of previous issues for example.
```
Comment (nickloestartup) [inline _posts/2017-05-22-Sherlock-Holmes-Day.md]: Not sure `for example` is needed on the end here?

File: _posts/2017-05-22-Sherlock-Holmes-Day.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+One of our testers had an interesting example of an issue they once investigated:
+*“I remember an occasion that a load of bookings had what looked like a flight number entered as vehicle details, something like "TGM18240", it took us ages to actually figure out what was happening, we investigated every bit of our code that we thought might be responsible. As it turned out, all these bookings had something else in common, the registration had been entered as "TBC", which wasn't odd as many of our customers enter "TBC", or, To Be Confirmed, as they haven't yet decided which car they will take to the airport. As it turned out, for some reason in our database, the registration plate "TBC" was assigned to a MAN lorry, and what looked like a flight number turned out to be the particular model, which translated as it's weight (18T) and the number of wheels or axles etc. Definitely a very weird one to investigate, and it provided an amusing anecdote to share round the office afterwards too!”*
```
Comment (nickloestartup) [inline _posts/2017-05-22-Sherlock-Holmes-Day.md]: I appreciate this is a quote, but we repeat `as it turned out`.

Plus I'm not sure what's meant by `MAN`?

**Comment by mattsaunby** [comment]
Thanks @nickloestartup - I've addressed your comments, would you mind taking another look please?

**Comment by nickloestartup** [comment]
Looks good to me 👍

**Pull request merged**